CLI/test/CLITest.cs

```diff
@@ -97,6 +97,9 @@ public void TestGenerate_NestedProcedure()
         
 
             [TestMethod]
+#if DOMAIN_CHECKER
+            [Ignore]//Because It will detect TypeDef Circularity two times.
```

Comment:
No this is not a bug, because of the code in CrossChecker that has been put to validate the Semantic Domain. These are CLI Tests so they use CrossChecker.CheckVariable, so we start thinking about : https://github.com/TypeCobolTeam/TypeCobol/issues/1278

Reply:
This fixed now, these tests pass

Reply:
Ok, so can you remove [Ignore] ?